.github/workflows/publish.yml

```diff
@@ -19,4 +19,4 @@ jobs:
       - run: yarn
       - run: yarn npm publish
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          YARN_NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Let's add a comment to this to point out that the GitHub docs don't actually show the correct name, "NODE_AUTH_TOKEN" does not work.